### DISCUSSION TITLE
Performance of loading multiple select elements

### DISCUSSION THREAD

**Discussion by nbwzx** [selectize/selectize.js#1904] [Q&A]
I did:

* [x] Search for if my issue has already been submitted
* [x] Make sure I'm reporting something precise that needs to be fixed
* [x] Give my issue a descriptive and concise title
* [x] Create a *minimal* working example on JsFiddle or Codepen
	(or gave a link to a demo on the Selectize docs)
* [x] Indicate *precise* steps to reproduce in *numbers* and the result,
	  like below

I have multiple(440) select elements on one page. It takes me about 2500ms to load them. Is there anything I can do to speed up it?

Steps to reproduce:

1. Go to [https://blddb.net/custom/edge.html](https://blddb.net/custom/edge.html), the source code is in [https://github.com/nbwzx/blddb/blob/main/assets/js/custom/edge.js](https://github.com/nbwzx/blddb/blob/main/assets/js/custom/edge.js)
2. Change the buffer (it will load all the select elements), which takes 2500ms.

Thanks.

**Comment by risadams** [comment]
v0.15x will have some performance improvements,  but I"m not quite sure I understand how to work with your example well to say for sure if it benefits you.

Could you please give me some guidance on what your example page is doing? and what values I should enter for "set up" position, etc.?

**Comment by nbwzx** [comment]
Sorry, I provided a wrong site. I have corrected it. [https://blddb.net/custom/edge.html](https://blddb.net/custom/edge.html)
